app/controllers/drafts_controller.rb

```diff
@@ -11,10 +11,9 @@ def show
 
   def create
     resource = load_resource
+    language_ids = [].push(data_attrs[:language_ids]).push(data_attrs[:language_id])
```

Comment:
This line isn't my favorite line ever. It's not exactly clear what you're expecting here.

Reply:
I would have approved this PR if you hadn't added this commit.

Reply:
So we can handle two types of input: `language_id: 1` or `language_ids: [1, 2, 3]`

Reply:
I will change it to only support the latter